code/modules/research/techweb/_techweb.dm

```diff
@@ -111,17 +125,17 @@
 	modify_point_list(l)
 
 /datum/techweb/proc/copy_research_to(datum/techweb/receiver, unlock_hidden = TRUE)				//Adds any missing research to theirs.
+	if(unlock_hidden)

```

Comment:
Why was this moved here ?

Reply:
Originally it was at the bottom of copy_research_to, so if you had a tech disk with JUST a hidden tech on it, you had to upload the techs contained within the disk twice before you were given that tech.